src/ordered_set_lang.ml

```diff
@@ -1,40 +1,59 @@
 open! Import
 
-type t = Sexp.Ast.t
+type expanded
+type unexpanded
```

Comment:
I think in general it's recommended to do:

```ocaml
type expanded = Expanded
type unexpanded = Unexpanded
```

to make it clear that the two types are distinct. I think here it's fine as the types are only used inside this module, but we might as well use the canonical thing.